README.md

```diff
@@ -1 +1,171 @@
-# d2-docker
\ No newline at end of file
+## Requirements
+
+-   Python >= 3.5 (with python3-distutils)
+-   Docker >= 18
+-   Docker compose >= 1.17
+
+## Usage
+
+### Start a DHIS2 instance
+
+Start a new container from a _dhis2-data_ base image:
+
+```
+$ bin/d2-docker start eyeseetea/dhis2-data:2.30-sierra
+```
+
+Notes:
+
+-   A d2-docker instance is composed of 4 containers: `dhis2-data` (database + apps), `dhis2-core` (tomcat + dhis.war), `postgis` (postgres with postgis support) and `nginx` (web server).
+-   By default, the image `dhis2-core` from the same organisation will be used, keeping the first part of the tag (using `-` as separator). For example: `eyeseetea/dhis2-data:2.30-sierra` will use core `eyeseetea/dhis2-core:2.30`. If you need a custom image to be used, pass the option `--core-image eyeseetea/dhis2-core:2.30-custom`.
+-   Once started, you can connect to the DHIS2 instance (`http://localhost:PORT`) where _PORT_ is the first available port starting from 8080. You can run many images at the same time, but not the same image more than once. You can specify the port with option `-p PORT`.
+-   Use option `--pull` to overwrite the local images with the images in the hub. Use option ``--detach` to run containers in the background.
```

Comment:
``detach``

Reply:
What do you mean?

Reply:
Sorry missing `

Reply:
Oh, my bad, I was concentrating on the spelling and didn't notice!